README.md

```diff
@@ -16,7 +16,15 @@ The **CLIP** model was originally proposed in  [Learning Transferable Visual Mod
 
 ## Prerequisites
 
-None
+To install the dependencies locally run 
+```
+pip install . 
+pip install -r tests/requirements.txt
```

Comment:
Why it is necessary to install `tests/requirements.txt`? This should be only for testing purpose.

```suggestion

```

Reply:
In case someone wants to run the test suite to verify correct installation. 
I would assume if a user clones the repository instead of installing it from the hub or via pip he or she intend to work on the code. Then running the test suite is relevant, what do you think?

Reply:
i see the points. It makes sense. `jina-commons` is an exception